protos/flow.proto

```diff
@@ -226,6 +236,11 @@ message PartitionRange {
   }
 }
 
+enum TableEngine {
+  CH_ENGINE_REPLACING_MERGE_TREE = 0;
```

Comment:
`UNKNOWN` should be 0, to differentiate between unset and explicit set

Reply:
unknown is -1

Reply:
can we make it explicit?

Reply:
what benefit does that bring? either unknown is de facto ReplacingMergeTree for backwards compat, or unknown is an error & we break existing mirrors if they resync

Reply:
I don't like the idea of the Golang unset value equaling anything because it can mask coding errors later

The issue with existing mirrors is valid, I'd fix it by doing a one-time "correction" like we've done before, but it's iffy and painful in practice

Reply:
embrace go's implicit nature, one time corrections are not something I want to make a habit of, they're error prone & put a burden on customers/support